portfolio.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="homestyle.css">
+  </head>
+  <body>
+	  <header class="navbar">
+		  <div id="NC"><a href="home.html">Nick Clifford</a> 
+		  </div>
+		  <div class="links">
+			  	<a id="aboutme" href="aboutme.html">About Me</a>
+			  	<a id="portfolio" href="portfolio.html">Portfolio</a>
+			 	<a id="contact" href="contact.html">Contact</a>
+		  </div>
+		  
+	  </header>
+	  <div class="portmain">
```

Comment:
Lines 19-21 aren't represented in CSS (i.e. no "portmain" or"porttitle") and the text "Things I have made..." don't show up